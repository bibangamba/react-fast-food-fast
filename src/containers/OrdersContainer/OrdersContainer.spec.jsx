import { shallow } from "enzyme";
import React from "react";
import OrdersView from "./OrdersContainer";

describe("OrdersView page test", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<OrdersView />);
  });

  it("matches snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
