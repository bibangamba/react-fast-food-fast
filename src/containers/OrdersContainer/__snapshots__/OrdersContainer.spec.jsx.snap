// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrdersView page test matches snapshot 1`] = `ShallowWrapper {}`;
